tests/unit/test_download.py

```diff
@@ -51,7 +51,10 @@ def _fake_session_get(*args, **kwargs):
 
 
 def test_user_agent():
-    PipSession().headers["User-Agent"].startswith("pip/%s" % pip.__version__)
+    user_agent = PipSession().headers["User-Agent"]
+
+    assert user_agent.startswith("pip/%s" % pip.__version__)
+    assert '"ci":true' in user_agent or '"ci":false' in user_agent
```

Comment:
I would prefer to test exhaustively here, some kind of parametrize per key exists/missing 🤔 as per this implementation the test would pass if someone writes either ``return True`` or ``return False`` inside looks like CI 🤔 such regressions would go without much notice